src/ccstudiodss/api.py

```diff
@@ -98,7 +100,7 @@ def __exit__(self, exc_type, exc_value, traceback):
             self.debug_session.target.disconnect()
             self.debug_server.stop()
         finally:
-            javabridge.kill_vm()
+            jpype.shutdownJVM()
```

Comment:
I hope that Session is a singleton.   JPype does not support starting and stopping of the JVM due to the design of the JVM.

Reply:
Thanks for pointing that out, but yes, I noticed.  That's the case in javabridge as well.  Setting that up properly to respect that is a thing I haven't done yet.